stories/GeoSearch.stories.js

```diff
@@ -130,3 +130,54 @@ stories
       filterProps,
     }
   );
+
+stories.addWithJSX('with unmount', () => {
+  class Example extends Component {
+    state = {
+      visible: true,
+    };
+
+    onToggle = () =>
+      this.setState(({ visible }) => ({
+        visible: !visible,
+      }));
+
+    render() {
+      const { visible } = this.state;
+
+      return (
+        <WrapWithHits
+          indexName="airbnb"
+          linkedStoryGroup="GeoSearch"
+          searchParameters={{ hitsPerPage: 20 }}
+        >
+          <Configure aroundLatLngViaIP />
+
+          <button onClick={this.onToggle} style={{ marginBottom: 15 }}>
+            {visible ? 'Unmout' : 'Mount'}
+          </button>
+
+          {visible && (
+            <Container>
+              <GoogleMapsLoader apiKey="AIzaSyCl2TTJXpwxGuuc2zQZkAlIkWhpYbyjjP8">
+                {google => (
+                  <GeoSearch google={google}>
+                    {({ hits }) => (
+                      <Fragment>
```

Comment:
It not needed, but it's more convenient because the array still need to have `key` on their elements. I can even render inside a `<div>` it doesn't matter much.